.devops/code-review-pipelines.yml

```diff
@@ -12,7 +12,7 @@ resources:
     - repository: pagopaEcommerceLocal
       type: github
       name: pagopa/pagopa-ecommerce-local 
-      ref: main
+      ref: allccp-lightcheck-flag
```

Comment:
main branch will be restored when https://github.com/pagopa/pagopa-ecommerce-local/pull/78 will be merged